src/sources/datadog_agent/traces.rs

```diff
@@ -174,7 +175,9 @@ fn convert_dd_tracer_payload(payload: ddtrace_proto::TracerPayload) -> Vec<Trace
             trace_event.insert("priority", trace.priority as i64);
             trace_event.insert("origin", trace.origin);
             trace_event.insert("dropped", trace.dropped_trace);
-            trace_event.insert("tags", Value::from(convert_tags(trace.tags)));
+            let mut span_tags = convert_tags(trace.tags);
```

Comment:
Is this really `span_tags`? Seems more like `trace_tags` to me since this function is mapping over trace events.